common/component_interface_tools/package.xml

```diff
@@ -15,9 +15,11 @@
   <depend>diagnostic_updater</depend>
   <depend>fmt</depend>
   <depend>rclcpp</depend>
+  <depend>service_log_checker</depend>
   <depend>tier4_system_msgs</depend>
   <depend>yaml_cpp_vendor</depend>
 
+  <test_depend>ament_cmake_ros</test_depend>
```

Comment:
This package uses `ament_cmake_auto`, so `ament_cmake_ros` is not needed if you use `ament_auto_add_gtest`. instead of `ament_add_ros_isolated_gtest`.